src/test_l1_dsHost.c

```diff
@@ -358,29 +364,35 @@ void test_l1_dsHost_negative_dsGetCPUTemperature(void) {
     float temperatureValue;
 
     // Step 01: dsGetCPUTemperature() Call without prior initialization
+    #ifdef ENABLE_ENHANCED_ERROR_CODE
     dsError_t result = dsGetCPUTemperature(&temperatureValue);
     UT_ASSERT_EQUAL(result, dsERR_NOT_INITIALIZED);
     UT_LOG("Step 01: Fetch CPU Temperature without initialization -> Expected: dsERR_NOT_INITIALIZED, Got: %d\n", result);
+    #endif
 
     // Step 02: dsHostInit() Initialize dsHost
     result = dsHostInit();
     UT_ASSERT_EQUAL(result, dsERR_NONE);
     UT_LOG("Step 02: Initialize dsHost -> Expected: dsERR_NONE, Got: %d\n", result);
 
     // Step 03: dsGetCPUTemperature() Call with NULL pointer
+    #ifdef ENABLE_ENHANCED_ERROR_CODE
```

Comment:
"dsERR_INVALID_PARAM" is an older, existing error code and not the enhanced version. Why we have commented out with enhanced error code?